### PR TITLE
Implement method for Thomas to say his hello

### PR DESCRIPTION
**Pull request by dahlgrenthomas**
This will give Thomas a hello in our java code.

### CODE REVIEW

**Review by jquist24 — DISMISSED** [review]
Looks good.

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String thomasSaysHello() {
+    return "Thomas says 'Hola!'\n";
+  }
+
```
Comment (DanteMillerDS) [inline src/main/java/hellos/Hellos.java:51]: good job!!!

**Review by DanteMillerDS — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -23,6 +23,7 @@ public String generateOutput() {
     builder.append(jeffSaysHello());
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
+    builder.append(thomasSaysHello());
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:30]: This is properly alphabetized.

File: src/main/java/hellos/Hellos.java [hunk 3]
```diff
@@ -36,6 +37,11 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+
+  private String thomasSaysHello() {
+    return "Thomas says 'Hola!'\n";
+  }
+
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:51]: This both follows the format of the other greetings and passes the tests. It should be ready for use!

**Review by day00090 — APPROVED** [review]

**Review by jquist24 — APPROVED** [review]
Looks not great.

**Pull request merged**